school_library/modules/create_books.rb

```diff
@@ -1,52 +1,53 @@
+# create_book.rb
 require_relative '../library/person'
 require_relative '../models/book'
 require_relative '../models/rental'
 require_relative '../models/classroom'
+require 'json'
 
 module CreateBook
```

Comment:
- [ ] Good job on saving all data into JSON files and loading all data from files on startup of the app, Kindly only save data in JSON files on the exit of your app instead of saving it after any adding operation as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_preserve_data.md#:~:text=On%20exit%20from%20your%20app%20save%20all%20the%20data%20(books%2C%20people%2C%20and%20rentals)%20into%20JSON%20files).